app/Domain/Tickets/Hxcontrollers/Milestones.php

```diff
@@ -48,4 +52,40 @@ public function showCard()
         $this->tpl->assign('milestone', $milestone);
 
     }
+
+    public function post($params)
+    {
+        $result = $this->ticketService->quickAddMilestone($params);
+        if (is_numeric($result)) {
+            $params['id'] = $result;
+            $this->tpl->setNotification($this->language->__('notification.milestone_created_successfully'), 'success');
+            $subject = $this->language->__('email_notifications.milestone_created_subject');
```

Comment:
The email notification part should actually just be in the quickAddMilestone Service. That way people will get notified event if the milestone is being created via api